### ISSUE TITLE
Stuck?

### ISSUE THREAD

**Issue by PeterSurda** [tip4commit/tip4commit#317]
New commits are not detected, payouts are not processed (stuck at Waiting for withdrawal). Also can you make the withdrawal limit customisable rather than fixed at 0.0010000?

**Comment by alexandrz** [comment]
I will check out tipping daemon when I get back to my laptop, it should
load all missing commits.

I currently process deposits manually because of backlog and higher fees.


On Feb 23, 2017 01:20, "Peter Šurda" <notifications@github.com> wrote:

New commits are not detected, payouts are not processed (stuck at Waiting
for withdrawal). Also can you make the withdrawal limit customisable rather
than fixed at 0.0010000?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tip4commit/tip4commit/issues/317>, or mute the thread

**Comment by PeterSurda** [comment]
Maybe you can setup some sort of monitoring? I could help you but I'm not familiar with ruby (and I hate the MVC pattern).

**Comment by PeterSurda** [comment]
Ok, loading commits works, but withdrawals still don't.

**Comment by alexandrz** [comment]
Yes, I still process it manually bacause of back log. I plan to work on setting up monitoring and migrating to a new wallet on Wednesday.

**Comment by PeterSurda** [comment]
Excellent, works fine now. Any chance of making the withdrawal threshold configurable, or perhaps even manual (user triggered)?

**Comment by nauadratti** [comment]
It's absolutely not clear how withdrawals are processed. I've got a tip for a commit, entered an address, now what? Just wait, it will be processed automatically? How long should I wait? Manual withdrawals would be much better.

**Comment by ProgVal** [comment]
Hi,

Payouts seem to be stuck again

**Comment by alexandrz** [comment]
Sent it. I am still processing payouts manually. Threshold is not configurable right now. I am not sure what to do, because sending bitcoins is expensive now. I am thinking of increasing the threshold while allowing users to withdraw money manually paying miners fees on their own. We might also implement http://faucetsystem.com/ support for smaller withdrawals.

**Comment by PeterSurda** [comment]
I thought I'd give you feedback, I personally wouldn't mind if the payouts are paused until the fees drop.
